libs/core/src/lib/select/select.component.ts

```diff
@@ -424,8 +424,7 @@ export class SelectComponent<T = any>
 
     /** @hidden */
     ngOnInit(): void {
-        this.ariaLabel = this.ariaLabel || this.placeholder;
-
+        this.ariaLabel = this.placeholder;
```

Comment:
If we have aria-label provided it will be overwritten with placeholder value, is it expected?

Reply:
I suppose this was sort of the previous behavior, when no ariaLabel was provided, so maybe technically a separate issue. Won't be overwritten by default given there is no default value

Reply:
I'll approve but also ask to remove this

Reply:
I am actually thinking it would make sense if the developer provides both an ariaLabel and a placeholder, that they'd want both read by the screenreader. So we won't want this line